lib/init/rc.shutdown

```diff
@@ -16,20 +16,18 @@ main() {
         dd count=1 bs=512 if=/dev/random of=/var/random.seed
     }
 
-    halt -w
```

Comment:
`halt` tells the system to cease all CPU activity though I'm unsure if it is needed as we send `TERM` regardless. I'm also fine with this change.

Reply:
ubase halt only has -p and -r switches

Reply:
This is fine to remove regardless as we only used `halt` to write the `wtmp` record.

```sh
-> halt --help
BusyBox v1.31.1 (2019-10-25 20:52:45 EEST) multi-call binary.

Usage: halt [-d DELAY] [-n] [-f] [-w]

Halt the system

        -d SEC  Delay interval
        -n      Do not sync
        -f      Force (don't go through init)
        -w      Only write a wtmp record
```

We really "halt" the machine ourselves below.